### PR TITLE
Tilesfix

### PR DESCRIPTION
**Pull request by voisardf**
@kalbermattenm : could you give it a very quick glance. Thanks

### CODE REVIEW

**Comment by kalbermattenm** [comment]
You use `node.attributes['format']` in `layerTree.js`, but in your Python files, you use `tile_format` as list key, thus I don't get how the ink between keys `format` and `tile_format` is made ?

I also don't really get what the changes in `model.py` have to do with the WMTS tiles...?

**Comment by voisardf** [comment]
models.py is a fix that ended up in this PR because I didn't branch from master by mistake. But for the two lines I didn't want to undo the action.

**Comment by voisardf** [comment]
@kalbermattenm: I cleaned up the differences between python and js - meaning I removed the parts that weren't used and generalized most of the hardcoded bits. For the rest issue #38 is needed.
thanks for reviewing

File: crdppf/templates/derived/init.js [hunk 1]
```diff
@@ -29,9 +29,9 @@ Crdppf.layers = [
             nbcols = len(layer)
         %>
             {
-            % for key in layer:
+            % for key in layer :
```
Comment (kalbermattenm) [inline crdppf/templates/derived/init.js]: I don't really get why you added a space everywhere before `:` in this file...

Python doesn't add a space before `:` only after...

**Comment by kalbermattenm** [comment]
OK, just a minor comment, but in my opinion #38 is a WIP...

**Comment by voisardf** [comment]
comment addressed. merging. thanks.

**Pull request merged**